src/plots/polar/layout_defaults.js

```diff
@@ -49,11 +51,7 @@ function handleDefaults(contIn, contOut, coerce, opts) {
         }
 
         var axIn = contIn[axName];
-        // Note: does not need template propagation, since coerceAxis is still
-        // based on the subplot-wide coerce function. Though it may be more
-        // efficient to make a new coerce function, then we *would* need to
-        // propagate the template.
```

Comment:
huh, so was this comment wrong all along, or did something change since I wrote this?

Reply:
That comment was only _partially_ correct/wrong.

We didn't have to propagate the templates for attributes getting coerced by the `coerce` function in the `polar/layout_defaults.js` scope, **but** we do need to for attributes getting coerced by `coerce2` like here:

https://github.com/plotly/plotly.js/blob/5c4094fc49e910da953b80b06f14e44332afd223/src/plots/cartesian/line_grid_defaults.js#L30-L32

Reply:
> but we do need to for attributes getting coerced by coerce2 like here

Ah... there it is, thanks.